src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -50,6 +51,21 @@ class DetectedLanguageSuitev4 extends TestBase with DataFrameEquality with TextK
     assert(iso(0).get(0).toString == "en" && iso(0).get(1).toString == "es" &&
       iso(1).get(0).toString == "fr" && iso(1).get(1).toString == "zh")
   }
+
+  test("Language Detection - Assert Confidence Score") {
+    val replies = detector.transform(df)
+      .select("output.result.name","output.result.confidenceScore")
+      .collect()
+
+    val confidence = replies.map(row => row.getList(1))
+    assert(confidence.indexOf(0) == -1)
+    assert(confidence.indexOf(1) == -1)
+  }
+
+  test("Language Detection - Print Schema") {
+    detector.transform(df).printSchema()
+    detector.transform(df).show()

```

Comment:
Should we have this as a test? this test does not contain any assertion. @HanLiMS